sunpy/visualization/animator/tests/test_mapsequence.py

```diff
@@ -0,0 +1,75 @@
+import os
+
+import pytest
+
+from astropy.visualization.wcsaxes import WCSAxes
+
+import sunpy.data.test
+import sunpy.map
+from sunpy.tests.helpers import figure_test
+from sunpy.visualization.animator import MapSequenceAnimator
+
+testpath = sunpy.data.test.rootdir
+
+
+@pytest.fixture
+def test_map_sequence():
+    return sunpy.map.Map(
+        os.path.join(
+            testpath, 'aia_171_level1.fits'), os.path.join(
+            testpath, 'mdi_fd_Ic_6h_01d.5871.0000_s.fits'), sequence=True)
+
+
+def test_construct_map_sequence_animator(test_map_sequence):
+    map_animator = MapSequenceAnimator(test_map_sequence)
+    assert isinstance(map_animator, MapSequenceAnimator)
+
+
+@figure_test
+def test_map_sequence_animator_wcs_simple_plot(test_map_sequence):
+    map_animator = MapSequenceAnimator(test_map_sequence)
+    map_animator._annotate_plot(0)
+    return map_animator.fig
+
+
+def test_axes(test_map_sequence):
+    map_animator = MapSequenceAnimator(test_map_sequence)
+    assert isinstance(map_animator.axes, WCSAxes)
+    start_img = map_animator.plot_start_image(map_animator.axes)
+    assert isinstance(start_img.axes, WCSAxes)
+
+
+@figure_test
+def test_map_sequence_animator_wcs_update_plot(test_map_sequence):
+    map_animator = MapSequenceAnimator(test_map_sequence)
+    map_animator.updatefig(1, map_animator.im, map_animator.sliders[0]._slider)
+    return map_animator.fig
```

Comment:
I feel like this is possible to test without a figure.

Reply:
I suppose, I'll make this change soon

Reply:
How exactly would the tests change? `updatefig` changes the current figures data and axes.